build.sh

```diff
@@ -522,6 +522,7 @@ if [ $FROM -le 1 ]; then
 			-DLLVM_BUILD_PATH="$LLVM_BUILD_PATH" \
 			-DLLVM_DIR=$LLVM_DIR \
 			-DCMAKE_INSTALL_PREFIX=$LLVM_PREFIX \
+			-DCMAKE_INSTALL_RPATH="\$ORIGIN/../lib" \
```

Comment:
I haven't used this CMake option yet: does RPATH get cleared on installing the binaries?

btw. you can avoid manually setting LD_LIBRARY_PATH by calling:
```shell
`symbiotic --dump-env-cmd`
```

Reply:
Yes, it is cleared by default. 

> By default if you don't change any RPATH related settings, CMake will
> link the executables and shared libraries with full RPATH to all used
> libraries in the build tree. When installing, it will clear the RPATH of
> these targets so they are installed with an empty RPATH.
> Source: https://gitlab.kitware.com/cmake/community/-/wikis/doc/cmake/RPATH-handling#default-rpath-settings

Reply:
I wonder why is the `\` before `$ORIGIN`? It results in setting the `RPATH` to `$ORIGIN/../lib` (the variable is not expanded). It is not what we want, right?

```
$ readelf -d install/llvm-10.0.0/bin/llvm-slicer
...
 0x000000000000001d (RUNPATH)            Library runpath: [$ORIGIN/../lib]
...
```

Reply:
Oh, sorry, forgot that $ORIGIN is a special variable. Ignore my previous comment...